kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/GwtWireServiceImpl.java

```diff
@@ -529,21 +529,21 @@ private void removeDeletedFromWireGraphProperty(final String pid) throws GwtKura
             // check if jObj is an empty JSON. It means all the existing
             // wire components need to be deleted
             if (length == 0) {
-                logger.info("Deleting Wire Component: PID -> " + componentPid);
+                logger.info("Deleting Wire Component: PID -> {}", componentPid);
                 configService.deleteFactoryConfiguration(componentPid, false);
                 continue;
             }
             boolean isFound = false;
             for (int i = 0; i < length; i++) {
                 final JsonObject jsonObject = jWireGraph.get(String.valueOf(i)).asObject();
                 final String component = jsonObject.getString("pid", null);
-                if (component.equalsIgnoreCase(componentPid)) {
+                if (component.equals(componentPid)) {
```

Comment:
Thanks Amit! @MMaiero Do you want to check for more instances of equalsIgnoreCase?